alien4cloud-yorc-plugin/src/main/resources/kubernetes/resources-matching-config.yml

```diff
@@ -12,3 +12,6 @@ matching_configurations:
       - docker_options: { equal: "" }
       - docker_run_cmd: { equal: "" }
       - gpuType: { equal: "" }
+  yorc.nodes.kubernetes.api.types.SimpleResource:
+    properties:
+      - size: { less_or_equal: 0 B }
```

Comment:
Ypu missed a new line at end of file

Reply:
> You missed a new line at end of file

Yes ! 
I will remove all this lines in fact because it is useless